### PR TITLE
Change variable names from wif_filename to priv_filename

### PR DESCRIPTION
**Pull request by ojalatodd**
Although .wif filenames were changed to .priv, variable
names were not changed.
This commit changes variable names from wif to priv.

Signed-off-by: Todd Ojala <todd@bitwise.io>

### CODE REVIEW

File: cli/sawtooth_cli/admin_command/keygen.py [hunk 1]
```diff
@@ -82,13 +82,13 @@ def do_keygen(args):
     pubkey = signing.generate_pubkey(privkey)
 
     try:
-        wif_exists = os.path.exists(wif_filename)
-        with open(wif_filename, 'w') as wif_fd:
+        priv_exists = os.path.exists(priv_filename)
+        with open(priv_filename, 'w') as wif_fd:
```
Comment (jjason) [inline cli/sawtooth_cli/admin_command/keygen.py]: `wif_fd` -> `priv_fd` and below when you `wif_fd.write`

File: cli/sawtooth_cli/keygen.py [hunk 2]
```diff
@@ -84,13 +84,13 @@ def do_keygen(args):
     pubkey = signing.generate_pubkey(privkey)
 
     try:
-        wif_exists = os.path.exists(wif_filename)
-        with open(wif_filename, 'w') as wif_fd:
+        priv_exists = os.path.exists(priv_filename)
+        with open(priv_filename, 'w') as wif_fd:
```
Comment (jjason) [inline cli/sawtooth_cli/keygen.py]: `wif_fd` -> `priv_fd` and below when `wif_fd.write`

File: extensions/arcade/sawtooth_battleship/battleship_cli.py [hunk 3]
```diff
@@ -227,23 +227,23 @@ def do_init(args, config):
...
+            if not os.path.exists(os.path.dirname(priv_filename)):
+                os.makedirs(os.path.dirname(priv_filename))
 
             privkey = signing.generate_privkey()
             pubkey = signing.generate_pubkey(privkey)
             addr = signing.generate_identifier(pubkey)
 
-            with open(wif_filename, "w") as wif_fd:
-                print("writing file: {}".format(wif_filename))
+            with open(priv_filename, "w") as wif_fd:
```
Comment (jjason) [inline extensions/arcade/sawtooth_battleship/battleship_cli.py]: `wif_fd` -> `priv_fd` and below when `wif_fd.write`

File: sdk/examples/xo_python/sawtooth_xo/xo_cli.py [hunk 4]
```diff
@@ -176,23 +176,23 @@ def do_init(args, config):
...
+            if not os.path.exists(os.path.dirname(priv_filename)):
+                os.makedirs(os.path.dirname(priv_filename))
 
             privkey = signing.generate_privkey()
             pubkey = signing.generate_pubkey(privkey)
             addr = signing.generate_identifier(pubkey)
 
-            with open(wif_filename, "w") as wif_fd:
-                print("writing file: {}".format(wif_filename))
+            with open(priv_filename, "w") as wif_fd:
```
Comment (jjason) [inline sdk/examples/xo_python/sawtooth_xo/xo_cli.py]: `wif_fd` -> `priv_fd` and below when `wif_fd.write`

File: extensions/arcade/sawtooth_battleship/battleship_cli.py [hunk 5]
```diff
@@ -227,23 +227,23 @@ def do_init(args, config):
 
     save_config(config)
 
-    wif_filename = config.get('DEFAULT', 'key_file')
-    if wif_filename.endswith(".priv"):
-        addr_filename = wif_filename[0:-len(".priv")] + ".addr"
+    priv_filename = config.get('DEFAULT', 'key_file')
+    if priv_filename.endswith(".priv"):
+        addr_filename = priv_filename[0:-len(".priv")] + ".addr"
```
Comment (jjason) [inline extensions/arcade/sawtooth_battleship/battleship_cli.py]: Should these be `addr_*` any more....or should they be `pub_*`???

File: cli/sawtooth_cli/admin_command/keygen.py (hunk 1 above)
Comment (ojalatodd) [inline cli/sawtooth_cli/admin_command/keygen.py]: Thanks, missed that. Fixed.

File: cli/sawtooth_cli/keygen.py (hunk 2 above)
Comment (ojalatodd) [inline cli/sawtooth_cli/keygen.py]: Fixed

File: extensions/arcade/sawtooth_battleship/battleship_cli.py (hunk 3 above)
Comment (ojalatodd) [inline extensions/arcade/sawtooth_battleship/battleship_cli.py]: fixed

File: sdk/examples/xo_python/sawtooth_xo/xo_cli.py (hunk 4 above)
Comment (ojalatodd) [inline sdk/examples/xo_python/sawtooth_xo/xo_cli.py]: Fixed

File: extensions/arcade/sawtooth_battleship/battleship_cli.py (hunk 5 above)
Comment (ojalatodd) [inline extensions/arcade/sawtooth_battleship/battleship_cli.py]: @jjason The ADDR file is being uses as an identifier, and is not a PUB file. I decided to leave as is. The pub file might serve fine as an identifier, but I wasn't sure. Let's leave as is. ;)

**Pull request merged**